server/src/test/java/io/spine/server/event/given/AbstractReactorTestEnv.java

```diff
@@ -0,0 +1,286 @@
+            return ImmutableList.copyOf(ordersServedLate);
+        }
+
+        private static Timestamp now() {
+            Instant currentInstant = Instant.now();
+            Timestamp result = Timestamps2.fromInstant(currentInstant);
+            return result;
+        }
+    }
+
+    /**
+     * Notifies customers and the delivery service when an order is ready to be served by emitting
+     * a respective event.
+     *
+     * <p>By default, uses SMS to notify customers.
+     */
+    public static class RestaurantNotifier extends AbstractEventReactor {
+
+        private final NotificationMethod notificationMethod;
+
+        public RestaurantNotifier(EventBus eventBus) {
+            super(eventBus);
+            notificationMethod = SMS;
+        }
+
+        @React
+        Pair<CustomerNotified, DeliveryServiceNotified>
+        notifyAboutOrder(OrderReadyToBeServed orderReady) {
+            Order order = orderReady.getOrder();
+            CustomerNotified customerNotified = notifyCustomer(order);
+            DeliveryServiceNotified deliveryNotified = notifyDelivery(order);
+            Pair<CustomerNotified, DeliveryServiceNotified> result = Pair.of(customerNotified,
+                                                                             deliveryNotified);
+            return result;
+        }
+
+        private CustomerNotified notifyCustomer(Order order) {
+            CustomerNotified result = CustomerNotified
+                    .newBuilder()
+                    .setOrder(order)
+                    .setNotificationMethod(notificationMethod)
+                    .build();
+            return result;
+        }
+
+        private static DeliveryServiceNotified notifyDelivery(Order order) {
+            String messageFormat = "Order %s is ready to be delivered.";
+            DeliveryServiceNotified result = DeliveryServiceNotified
+                    .newBuilder()
+                    .setMessage(format(messageFormat, order.getOrderId()))
+                    .setOrder(order)
+                    .build();
+            return result;
+        }
+    }
+
+    /** Obtains an event that signgifes that some order is ready to be served. */
+    public static OrderReadyToBeServed someOrderReady() {
+        OrderReadyToBeServed result = OrderReadyToBeServed
+                .newBuilder()
+                .setOrder(someOrder())
+                .build();
+        return result;
+    }
+
+    /** Throws an exception whenever an order is ready. */
+    public static class FaultyNotifier extends AbstractEventReactor {
+
+        public FaultyNotifier(EventBus eventBus) {
+            super(eventBus);
+        }
+
+        @SuppressWarnings("NewExceptionWithoutArguments") /* Does not matter for this test case. */
+        @React
+        Pair<CustomerNotified, DeliveryServiceNotified>
+        notifyAboutOrder(OrderReadyToBeServed orderReady) {
+            throw new RuntimeException();
```

Comment:
Can't you just add a message to the exception and remove the suppression?

Just in case, I don't understand why you throw the exception. Let's discuss.